src/main.rs

```diff
@@ -66,6 +66,8 @@ fn main() {
         let args2 = args
             .iter()
             .map(|s| (*s).to_string())
+            // Note: if running without rustup, comment the next line out
+            // and manually pass in the rust compiler dir via --sysroot
```

Comment:
In clippy they have a more sophisticated implementation for this (https://github.com/rust-lang/rust-clippy/blob/master/src/driver.rs#L308). If it bothers you to comment this out, maybe we can copy from there (perhaps a different PR)